pages/docs/reference/using-gradle.md

```diff
@@ -610,8 +610,10 @@ It is also possible to configure all Kotlin compilation tasks in the project:
 <div class="sample" markdown="1" mode="groovy" theme="idea" data-lang="groovy">
 
 ```groovy
-tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).configureEach {
-    kotlinOptions { //... }
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
```

Comment:
Note that the snippet that uses the FQN inline is a bit easier to copy-paste, so I'm not sure we should switch to using imports.